lib/scenic/adapters/postgres.rb

```diff
@@ -84,8 +86,10 @@ def drop_materialized_view(name)
       # @return [void]
       def refresh_materialized_view(name, concurrently: false)
         if concurrently
+          raise StandardError, "In order to refresh materialized views concurrently you must use PostgreSQL version 9.4 or newer" unless version_number >= 940
```

Comment:
We also try to avoid both `unless` and "conditional surprises" at the end of lines.